OWNERS_ALIASES

```diff
@@ -70,6 +70,7 @@ aliases:
     - xiangpengzhao
     - zacharysarah
     - zparnold
+    - rajeshdeshpande02
```

Comment:
Hi. This list is ordered alphabetically. It will be easiest if you add the new entry following that sort order.

Reply:
Hi @sftim, Thank you for review. I have made suggested changes. Can you please have a look.